sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1110,6 +1110,37 @@ pi_result piGetDeviceAndHostTimer(pi_device Device, uint64_t *DeviceTime,
   return pi2ur::piGetDeviceAndHostTimer(Device, DeviceTime, HostTime);
 }
 
+pi_result piextEnablePeerAccess(pi_device command_device,
+                                pi_device peer_device) {
+
+  std::ignore = command_device;
+  std::ignore = peer_device;
+
+  return PI_ERROR_PLUGIN_SPECIFIC_ERROR;
```

Comment:
I've made this work now. I had removed it because suddenly I got errors about `UR_ERROR_PLUGIN_SPECIFIC_ERROR` which had worked before. I've got round this by setting `UR_RESULT_ERROR_INVALID_VALUE`.

Reply:
In L0, PeerAccess is always enabled, so I dont think we should return SPECIFIC_ERROR.

What this function should do is to calle zeDeviceCanAccessPeer, and if it returns true, then we return SUCCESS here, and if not, then we return an error.

piextDisablePeerAccess I agree it should return an error, but maybe **UR_RESULT_ERROR_INVALID_OPERATION** is more appropriate?

Reply:
Also, this logic should go directly to the UR L0 adapter code. this file pi_level_zero.cpp should not contain any logic by only wrappers to the UR L0 adapter.

Reply:
ah, we need https://github.com/oneapi-src/unified-runtime/issues/68